_data/accessibility-tests/button-group.yml

```diff
@@ -0,0 +1,85 @@
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Keyboard actions can access all button group functionality.
+    summary_additional: You can navigate back and forth between buttons in the button group and perform all other functions using only your keyboard.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Button group does not trap keyboard focus.
+    summary_additional: When you use your keyboard to tab back and forth in a button group, you can easily move in and out to get to other content.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.2
+  - summary: Button focus follows logical content order.
+    summary_additional: When you tab through the button group with a keyboard, the focus stays in order of the content as intended.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.3
+  - summary: Focus indicator is clearly visible on button group.
+    summary_additional: When you use a keyboard to tab into the buttons in a group, the selected button has a visible outline or other clear indication of focus.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+  - summary: The focus indicator has proper size, perimeter, contrast and space between buttons.
+    summary_additional: When you use a keyboard and view the focus indicator on a button in a group, you can tell it is at least the area of 2 CSS pixel unfocused and has 3:1 contrast between focused and unfocused button states.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.13
+# Screen reader tests
+  - summary: Screen reader announces buttons in group.
+    summary_additional: When you use a screen reader to navigate into a button group, the button labels you hear match the visual labels on the screen.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.1
+  - summary: Buttons are announced in the same order that they appear on the page.
+    summary_additional: When you navigate to the first button in a group using a screen reader and tab back and forth between buttons, the announced button labels match the visual presentation.
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: Screen reader announces labels and state of each element.
+    summary_additional: When you use a screen reader and tab through a button group, you hear the role "button" or "link" and that it is part of a group. You will also hear whether the button or link has been visited or not.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 4.1.2
```

Comment:
**Todo:** There was a recent update to the summary on the checklist:

> **Screen reader announces labels and state of each element.**
> When you use a screen reader and tab through a button group, you hear the element "button" or "link" and that it is part of a group. You will also hear if a link has been visited or not.
>

```suggestion
  - summary: Screen reader announces labels and state of each element.
    summary_additional: When you use a screen reader and tab through a button group, you hear the element "button" or "link" and that it is part of a group. You will also hear if a link has been visited or not.
    test_status: conditional
    test_type: screen_reader
    version_tested: 3.8.1
    wcag_criterion: 4.1.2
```

Reply:
Resolved in [a1d0aa1](https://github.com/uswds/uswds-site/pull/3009/commits/a1d0aa157e9fe707bf0aa30a67e7b4e1caa66686)